src/operation/mod.rs

```diff
@@ -183,42 +186,6 @@ impl<T: CommandBody> Command<T> {
     }
 }
 
-/// Trait modeling the behavior of a command response to a server operation.
-pub(crate) trait Response: Sized {
```

Comment:
This trait was no longer needed now that we extract these fields manually via the `RawDocument` API.